### PR TITLE
Merge `.coco.` and `.ozar.` files (round 10)

### PR DESCRIPTION
**Pull request by adamkecskes**
New round of coco/ozar merge.

87 pairs left.

Zipper merge scrip was run, but you can check all the modifications separately if you check the commits included in this PR.

### CODE REVIEW

File: app/styles/user/certificates-view.ozar.sass [hunk 1]
```diff
@@ -119,34 +142,158 @@
...
+          left: 0
 
       .hero-signature-label
         text-transform: uppercase
         font-size: 9pt
         line-height: 18pt
         text-align: left
+
+    @if $is-codecombat
+      .product-codecombat & .teacher-info
```
Comment (mrfinch) [inline app/styles/user/certificates-view.ozar.sass:240]: have we added this product-codecombat class somewhere or it's getting added automatically?

File: app/views/artisans/StudentSolutionsView.ozar.coffee [hunk 2]
```diff
@@ -181,7 +195,7 @@ module.exports = class StudentSolutionsView extends RootView
       # aether.problems?
     if lang is 'javascript'
       try
-        ast = parser(src)
+        ast = @parser(src)
```
Comment (mrfinch) [inline app/views/artisans/StudentSolutionsView.ozar.coffee]: old bug caught?

**Review by mrfinch — COMMENTED** [review]
we are getting close to the unification!!! Thanks for all the efforts!!

File: app/styles/user/certificates-view.ozar.sass (hunk 1 above)
Comment (adamkecskes) [reply app/styles/user/certificates-view.ozar.sass:240]: @mrfinch its added in CocoView constructor: https://github.com/codecombat/codecombat/blob/master/app/views/core/CocoView.coffee#L62

**Pull request merged**